### PR TITLE
Executable Tutorial Submission

### PR DESCRIPTION
**Pull request by HexuL**
# Assignment Submission

## Title

Using Skaffold for Local Kubernetes Development and CI/CD

## Names and KTH ID

  - Hexu Li (hexu@kth.se)
  - Tianning Liang (tianning@kth.se)

## Deadline

- Task 3

## Category

- Executable tutorial

## Description

We will introduce how to use Skaffold for automating build, push, and deployment in a local Kubernetes cluster (Minikube). We plan to create a simple application, write its Dockerfile and Kubernetes manifest, and use Skaffold to streamline development. 

**Relevance**

Skaffold can simplify the CI/CD process and speed up deployment, making it easier to modify and debug configurations.

**Submission**

The Git Repo of the tutorial: https://github.com/HexuL/Executable-Tutorial

The tutorial: https://killercoda.com/executable-tutorial

### CODE REVIEW

File: contributions/executable-tutorial/hexu-tianning/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Using Skaffold for Local Kubernetes Development and CI/CD.
 
 ## Deadline
 
-- Task 3
+- Task 2
```
Comment (sofiabobadilla) [inline contributions/executable-tutorial/hexu-tianning/README.md]: Originally you choose Task 3, please maintain that.

**Comment by sofiabobadilla** [comment]
Great.
I'll merge the submission.

**Comment by sofiabobadilla** [comment]
HI @HexuL 
I'm running your tutorial and I keep getting:
"one or more valid Kubernetes manifests are required to run skaffold"

I'm on the demo folder, and already created the deployment.yml file.

Any idea of what could be the error?

**Comment by Tian0602** [comment]
> HI @HexuL I'm running your tutorial and I keep getting: "one or more valid Kubernetes manifests are required to run skaffold"
> 
> I'm on the demo folder, and already created the deployment.yml file.
> 
> Any idea of what could be the error?
@sofiabobadilla 
Hi! I followed the tutorial again and didn't encounter the error. Since the "Kubernetes manifests" should refer to skaffold.yaml, did you face the error in Initialize Skaffold:step 2 or did you create skaffold.yaml successfully?

**Comment by sofiabobadilla** [comment]
I tried again and it is running.
I 'll add my comments on canvas :)

**Pull request merged**